closed/get_j9_source.sh

```diff
@@ -39,6 +39,10 @@ usage() {
 	echo "  -omr-branch       the OpenJ9/omr git branch: openj9"
 	echo "  -omr-sha          a commit SHA for the omr repository"
 	echo "  -omr-reference    a local repo to use as a clone reference"
+	echo "  -openjceplus-repo the OpenJCEPlus repository url: git@github.ibm.com:JavaSecurity/OpenJCEPlus-with-GSKIT.git"
```

Comment:
I suggest there should be no default for these options, but they certainly should not be IBM internal.